crates/egui/src/containers/window.rs

```diff
@@ -426,6 +426,12 @@ impl<'open> Window<'open> {
             0.0
         };
 
+        let viewport_rect = ctx.input(|i| i.viewport().inner_rect.unwrap_or(Rect::EVERYTHING));
```

Comment:
I think it makes more sense to constrain it to `area.constrain_rect`

Reply:
you're right. May be, It's what I wanted.

Reply:
It's what I wanted right.

```
        let viewport_rect = ctx.input(|i| i.viewport().inner_rect.unwrap_or(Rect::EVERYTHING));
        let max_width = viewport_rect.width();
        let max_height = viewport_rect.height() - title_bar_height;
```

using `area.constrain_rect` does not limit window from continuing to grow.